### ISSUE TITLE
Latest Version Update

### ISSUE THREAD

**Issue by FocusZTV** [twitch4j/twitch4j#132]
Hello,

How am I supposed to see the latest version of the API? Neither the README or documentation have been updated with the latest version number.

I also can't seem to use version 1.0.0-alpha.24, as it is not found, and gives me an error in my pom.xml, and when I try to run the application it fails on getting the classes in the API.

Thanks

**Comment by condolent** [comment]
You can find builds of the development branch [here](https://oss.jfrog.org/artifactory/libs-release/com/github/twitch4j/twitch4j/).

This requires you to use the jcenter repo:
```xml
<repository>
	<id>jcenter-snapshot</id>
	<url>https://oss.jfrog.org/artifactory/libs-release/</url>
</repository>
```

**Comment by FocusZTV** [comment]
> You can find builds of the development branch [here](https://oss.jfrog.org/artifactory/libs-release/com/github/twitch4j/twitch4j/).
> 
> This requires you to use the jcenter repo:
> 
> ```
> <repository>
> 	<id>jcenter-snapshot</id>
> 	<url>https://oss.jfrog.org/artifactory/libs-release/</url>
> </repository>
> ```

Ok, thanks, but I still can't use the latest version even when I use that repository. The most recent version I can use is 1.0.0-alpha.22, and I need the new fixes in version 1.0.0-alpha.24.

**Comment by condolent** [comment]
Then you need to update your maven dependencies.

[If you're using eclipse](https://stackoverflow.com/a/2556197/5424516)
